pkg/playground/playground.go

```diff
@@ -22,11 +22,16 @@ const (
 	contentTypeTextJavascript = "text/javascript"
 )
 
+// Config is the configuration Object to instruct ConfigureHandlers on how to setup all the http Handlers for the playground
 type Config struct {
+	// URLPrefix is a prefix you intend to put in front of all handlers
 	URLPrefix                   string
+	// PlaygroundURL is the URL where the playground website should be hosted
 	PlaygroundURL               string
```

Comment:
I suppose `PlaygroundURL`, `GraphqlEndpointURL` is not a URLs, but a path
maybe name them as `PlaygroundPath`?